trino/integration_test.go

```diff
@@ -82,11 +83,17 @@ func TestMain(m *testing.M) {
 			if *trinoImageTagFlag == "" {
 				*trinoImageTagFlag = "latest"
 			}
+			exposedPorts := []string{
+				"8080/tcp",
+				"8081/tcp",
+			}
+
 			resource, err = pool.RunWithOptions(&dt.RunOptions{
-				Name:       name,
-				Repository: "trinodb/trino",
-				Tag:        *trinoImageTagFlag,
-				Mounts:     []string{wd + "/etc:/etc/trino"},
+				Name:         name,
+				Repository:   "trinodb/trino",
+				Tag:          *trinoImageTagFlag,
+				Mounts:       []string{wd + "/etc:/etc/trino"},
+				ExposedPorts: exposedPorts,
```

Comment:
Why do we need to expose ports? This will cause conflicts if anything already uses these ports (other containers).

Reply:
@nineinchnick - These are container ports and mapped to some random ports on host m/c, they wont't conflict. I have setup 2 ports, 8080 is without authentication and 8081 is with jwt authentication.